php/libraries/NDB_Config.class.inc

```diff
@@ -570,9 +572,7 @@ class NDB_Config
             $nextLevel = NDB_Config::getMenuTabs($thisRow['ID']);
 
             if (!empty($nextLevel)) {
-                if (NDB_Config::checkMenuPermission($nextLevel[0]['ID'])) {
```

Comment:
I think @ridz1208 convinced me that the logic for removing this check is sound, it's always an `if (true)` check because of the array_filter line, but he wants to move the for loop into the `if` since the `else` always returns to make this more readable.